### PR TITLE
PBI-57: Button

### PR DESCRIPTION
**Pull request by ferryhieronimus**
## Describe your changes

Added Button component

## Issue ticket number and link

ADU-53: https://linear.app/adudu/issue/ADU-57/button

### CODE REVIEW

**Review by edutjie — APPROVED** [review]
LGTM

**Review by valenciusap17 — APPROVED** [review]
LGTM 🔥

**Comment by FreeJ1nG** [comment]
LGTM, Merging now

File: src/components/atoms/button/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    const { container } = render(
+      <Button variant="default">Default Button</Button>,
+    )
+    const buttonElement = container.querySelector('button')
+    expect(buttonElement).toHaveClass(
+      'bg-primary',
+      'text-primary-foreground',
+      'hover:bg-primary/90',
+    )
+  })
```
Comment (FreeJ1nG) [inline src/components/atoms/button/index.test.tsx:39]: Now that i see this, I think this should make use of snapshot testing

```ts
expect(buttonElement.className).toMatchInlineSnapshot()
```

Comment (ferryhieronimus) [reply src/components/atoms/button/index.test.tsx:39]: Why?

Comment (FreeJ1nG) [reply src/components/atoms/button/index.test.tsx:39]: If the className/theme changes in the future, we can just run snapshot testing with the `-u` option

**Pull request merged**